ledger/sandbox-on-x/BUILD.bazel

```diff
@@ -137,6 +158,43 @@ da_scala_binary(
     ],
 )
 
+da_scala_binary(
+    name = "conformance-test-oracle-bin",
+    main_class = "com.daml.ledger.sandbox.MainWithEphemeralOracleUser",
+    visibility = ["//visibility:public"],
+    runtime_deps = [
+        "@maven//:com_oracle_database_jdbc_ojdbc8",
+    ],
+    deps = [
+        ":sandbox-on-x-test-lib",
+    ],
+)
+
+da_scala_binary(
+    name = "conformance-test-postgres-bin",
+    main_class = "com.daml.ledger.sandbox.MainWithEphemeralPostgresql",
+    visibility = ["//visibility:public"],
+    runtime_deps = [
+        "@maven//:com_oracle_database_jdbc_ojdbc8",
```

Comment:
Why is Oracle driver needed for Postgres?

Reply:
it definitely does not :+1: 
funny thing we do not need this for oracle neither: these deps are creeping in as build deps already, removed that as well